docs/breaking-changes.md

```diff
@@ -30,6 +30,55 @@ webContents.setWindowOpenHandler((details) => {
 })
 ```
 
+### Removed: BrowserWindow `scroll-touch-*` events
+
+The `scroll-touch-begin`, `scroll-touch-end` and `scroll-touch-edge` events on
+BrowserWindow have been removed.
+
+```js
+// Removed in Electron 22.0
+win.on('scroll-touch-begin', scrollTouchBegin)
+win.on('scroll-touch-edge', scrollTouchEdge)
+win.on('scroll-touch-end', scrollTouchEnd)
+
+// Replace with
+win.webContents.on('input-event', (_, event) => {
```

Comment:
It's a bug, thanks for catching! I was initially confused about the caseness of these types because the code accepts any case for types for some reason, so I'd expressed all the event types as lowercase since that's how some of the tests were written. I realised later that they should be camelCased when we produce them, but forgot to update these examples.